src/Components/Nav/Nav.scss

```
@@ -0,0 +1,202 @@
+@mixin borderZero {
+  border: 0;
+  outline: none;
+}
+.Nav {
+  padding-top: 120px;
+  .navWrapper {
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
```

```suggestion
.Nav {
  padding-top: 120px;
  
  .navWrapper {
    position: fixed;
    top: 0;
    left: 0;
    right: 0;
```
- left, right 속성을 모두 0 주고 있어서 width 100%는 불필요해보입니다!
- 새로운 셀렉터가 시작할 때는 항상 한줄씩 띄워주세요.